app/products/calls/screens/participants_list/pill.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useMemo} from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+
+import {useTheme} from '@context/theme';
+import {changeOpacity} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+type Props = {
+    text: string | number;
+}
+
+const getStyleSheet = ({theme}: { theme: Theme }) => {
```

Comment:
please use `makeStyleSheetForTheme` instead of memoizing the style in the component